README.md

```diff
@@ -56,11 +56,18 @@ yarn ts-node ./scripts/referrerUserCount.ts --protocol Beefy --referrerIds app1
 
 ## Contracts
 
-This repository contains the contract(s) necessary to support the Mobilestack Funding Layer.
-
-Registry contract address: 0xBa9655677f4E42DD289F5b7888170bC0c7dA8Cdc
-
-Owner safe: 0xfC95675a6bB93406C0CbBa9403a084Dd8D566F06
+This repository contains the contract(s) necessary to support the Divvi protocol v0.
+
+| Network   | Registry Contract                     | Multisig Address                      |
+|-----------|---------------------------------------|---------------------------------------|
+| Arbitrum  | [`0xBa9655677f4E42DD289F5b7888170bC0c7dA8Cdc`](https://arbiscan.io/address/0xBa9655677f4E42DD289F5b7888170bC0c7dA8Cdc) | [`0xfC95675a6bB93406C0CbBa9403a084Dd8D566F06`](https://app.safe.global/home?safe=arb1:0xfC95675a6bB93406C0CbBa9403a084Dd8D566F06) |
+| Base      | [`0xBa9655677f4E42DD289F5b7888170bC0c7dA8Cdc`](https://basescan.org/address/0xba9655677f4e42dd289f5b7888170bc0c7da8cdc) | [`0xfC95675a6bB93406C0CbBa9403a084Dd8D566F06`](https://app.safe.global/home?safe=base:0xfC95675a6bB93406C0CbBa9403a084Dd8D566F06) |
+| Berachain | [`0x5Ac8EB3Bfcb40daF6a209779Ac8643075222285f`](https://berascan.com/address/0x5Ac8EB3Bfcb40daF6a209779Ac8643075222285f) | [`0xBdae4f72Dbc36Eef3772F5cdDFB9DCbD88bF22BD`](https://safe.berachain.com/home?safe=berachain:0xBdae4f72Dbc36Eef3772F5cdDFB9DCbD88bF22BD) |
+| Celo      | [`0xBa9655677f4E42DD289F5b7888170bC0c7dA8Cdc`](https://celoscan.io/address/0xba9655677f4e42dd289f5b7888170bc0c7da8cdc) | [`0xfC95675a6bB93406C0CbBa9403a084Dd8D566F06`](https://app.safe.global/home?safe=celo:0xfC95675a6bB93406C0CbBa9403a084Dd8D566F06) |
+| Ethereum  | [`0xBa9655677f4E42DD289F5b7888170bC0c7dA8Cdc`](https://etherscan.io/address/0xBa9655677f4E42DD289F5b7888170bC0c7dA8Cdc) | [`0xfC95675a6bB93406C0CbBa9403a084Dd8D566F06`](https://app.safe.global/home?safe=eth:0xfC95675a6bB93406C0CbBa9403a084Dd8D566F06) |
+| Optimism  | [`0xBa9655677f4E42DD289F5b7888170bC0c7dA8Cdc`](https://optimistic.etherscan.io/address/0xba9655677f4e42dd289f5b7888170bc0c7da8cdc) | [`0xfC95675a6bB93406C0CbBa9403a084Dd8D566F06`](https://app.safe.global/home?safe=oeth:0xfC95675a6bB93406C0CbBa9403a084Dd8D566F06) |
+| Polygon PoS | [`0xBa9655677f4E42DD289F5b7888170bC0c7dA8Cdc`](https://polygonscan.com/address/0xBa9655677f4E42DD289F5b7888170bC0c7dA8Cdc) | [`0xfC95675a6bB93406C0CbBa9403a084Dd8D566F06`](https://app.safe.global/home?safe=matic:0xfC95675a6bB93406C0CbBa9403a084Dd8D566F06) |
+| Vana      | [`0x0`](https://vanascan.io/address/0x0) | [`0xFc273c89EE7570Fa154091ca6068683Af2293cF7`](https://safe.vana.org/home?safe=vana:0xFc273c89EE7570Fa154091ca6068683Af2293cF7) |
```

Comment:
Is the registry contract really deployed at `0x0`? :eyes:

Reply:
`0x2C9403cdCfb51E03fa04131cEBF9E686a7a48A30`

Reply:
it was just a placeholder